### DISCUSSION TITLE
I get error after updated @tanstack/solid-virtual to "^3.0.1"

### DISCUSSION THREAD

**Discussion by F1shez** [TanStack/virtual#638] [Q&A]
When update @tanstack/solid-virtual "3.0.0-beta.35" to "^3.0.1", i get zero array from getVirtualItems(). 
Example: https://codesandbox.io/p/devbox/tanstack-solid-virtual-example-dynamic-57vs8v?file=%2Fsrc%2Fmain.tsx%3A79%2C20
You can open and update @tanstack/solid-virtual in this example, and you will get error.

**Comment by F1shez** [comment]
i can add check length array before rendering, but is this the correct behavior?

**Comment by AIvan83** [comment]
I have the same thing, I found out that the error appeared after **3.0.0-beta.62**
